src/main/java/com/faforever/client/mod/ModServiceImpl.java

```diff
@@ -181,7 +182,12 @@ public void loadInstalledMods() {
   @SneakyThrows
   @Override
   public CompletableFuture<Void> downloadAndInstallMod(String uid) {
-    return fafService.getMod(uid).thenAccept(mod -> downloadAndInstallMod(mod, null, null));
+    return fafService.getModVersion(uid)
+        .thenCompose(mod -> downloadAndInstallMod(mod, null, null))
+        .exceptionally(throwable -> {
+          logger.warn("Sim mod was not installed", throwable);
```

Comment:
`Sim mod could not be installed` (otherwise, it sounds like "it wasn't but now it is")